Library/src/main/java/com/pusher/android/PusherAndroid.java

```diff
@@ -109,6 +109,11 @@ public PresenceChannel getPresenceChannel(String channelName) {
         return this.pusher.getPresenceChannel(channelName);
     }
 
+    /**
+     * @return the client for push notifications
+     * @deprecated this feature is being decommissioned, please use Pusher Beams instead
```

Comment:
I think that we should clarify that we'll remove this method in the next version.

Reply:
Aren't all methods marked as deprecated a signal that they are going to be removed eventually?

Also, I don't know when we are releasing the next version of this library as it doesn't make sense to exist without the push notifications functionality -- you can just use the `pusher-websocket-java`.

Reply:
In the Java world, some "deprecated" APIs [are still there](https://docs.oracle.com/javase/8/docs/api/deprecated-list.html) after 20 years!

But I agree with you about releasing the next version. Although I think that the work on end-to-end encryption will require a new release.